.gitignore

```diff
@@ -10,14 +10,13 @@ build/
 *.coverprofile
 *.test
 *.orig
-*/vendor
 vendor
 .DS_Store
 .bak
 .idea/
 .vscode/
-celestia
-cel-shed
-cel-key
+/celestia
+/cel-shed
+/cel-key
```

Comment:
Why do we need to track the root dir only(where .gitignore is located)? If someone compiles a binary, not in the root, but somewhere deeper in the path, it should not be included in git history either, imo.

Reply:
Because the binary file has no extension name, git can't distinguish file and folder. When I add new files to cmd/celestia, git can't track it. If we limit celestia binary in the root folder, I feel it's better to apply to other binaries too, easier for understanding. Each coder may have their own configuration so we can only focus on the default config. They could have their local version when they need it. According to the current default config, the celestia binary isn't in the root folder, but I don't know if there are coders who still prefer it, we don't need to bother them, just provide backward compatibility. For the 'build' folder, I see a lot of projects put scripts about the building processes in this folder, and use some like 'target' as output, and I don't know if we will import code from other projects later, then limiting our build folder in root may get better compatibility.

Reply:
@Wondertan What do you prefer, change the one caused problem and keep others untouched?

Reply:
Sounds reasonable.